docs/03-endpoints/api-admin/projects.md

```diff
@@ -813,6 +813,42 @@ Example response:
 }
 ```
 
+### Set Restricted View Settings
```

Comment:
Please also update the overview table at the beginning of the file, for consistency